torch/csrc/distributed/rpc/process_group_agent.h

```diff
@@ -77,6 +77,12 @@ class ProcessGroupAgent : public RpcAgent {
       override;
 
  private:
+  using steady_clock_time_point =
+      std::chrono::time_point<std::chrono::steady_clock>;
+
+  static constexpr steady_clock_time_point kInfiniteTimeoutTimePoint =
```

Comment:
Should this go in rpc_agent.h so that other RpcAgents can have access to them?

Reply:
I don't think so. This watch dog thread and using a const time to represent infinity are all specific to ProcessGroupAgent only.

Notice this is not the "Default RPC timeout".